apps/epl/src/epl_traffic.erl

```diff
@@ -242,7 +242,11 @@ push_region(Name, Vizceral) ->
     push_region(Name, #{}, Vizceral).
 
 push_region(Name, Additional, Vizceral) ->
+    %% We assume that INTERNET is entryNode for every region
+    %% Vizceral has backward compatibility and in region view INTERNET node is
+    %% default entryNode if other isn't specified
     A = maps:merge(#{
+                      %% entryNode => <<"INTERNET">>,
```

Comment:
At first I was trying to set `entryNode` explicit but this caused Vizceral crashes. I havr no idea why thats happend but I'm pro explicity entryNode than relaying on default value